### PR TITLE
Log query requests and query responses

### PR DESCRIPTION
**Pull request by jzonthemtn**
For #1 , here's an example of how the query requests and query responses are indexed with the `queryResponseHitIds`, `queryResponseId`, `queryId`, and the `query` itself.

These are queries against the store itself so that's why the index is `.awesome_queries`.

```
{
  "took": 791,
  "timed_out": false,
  "_shards": {
    "total": 1,
    "successful": 1,
    "skipped": 0,
    "failed": 0
  },
  "hits": {
    "total": {
      "value": 36,
      "relation": "eq"
    },
    "max_score": 1,
    "hits": [
      {
        "_index": ".awesome_queries",
        "_id": "29hrso0BJblqvzTXX2AW",
        "_score": 1,
        "_source": {
          "queryResponseHitIds": [
            "0",
            "1",
            "2",
            "3",
            "4",
            "5",
            "6",
            "7",
            "8",
            "9"
          ],
          "queryResponseId": "fc596fbd-79e8-440d-8607-f45db7df915c",
          "query": "{}",
          "queryId": "b3120e62-35bd-46b4-b36b-4ed4e574b1d3"
        }
      },
      {
        "_index": ".awesome_queries",
        "_id": "2dhrso0BJblqvzTXIGCX",
        "_score": 1,
        "_source": {
          "queryResponseHitIds": [
            "0",
            "1"
          ],
          "queryResponseId": "d4a21117-5a8d-42f6-af92-c301016d1c82",
          "query": "{}",
          "queryId": "28b2e0bc-dc75-4620-8c2b-5aa466cafe4b"
        }
      }
    ]
  }
}

```

### CODE REVIEW

**Comment by epugh** [comment]
I wonder if we need the equivalent of "techproducts" to make it clearer what is going on?  So less "self referential" ;-).   THis is cool to see...

**Comment by jzonthemtn** [comment]
W

> I wonder if we need the equivalent of "techproducts" to make it clearer what is going on? So less "self referential" ;-). THis is cool to see...

We definitely do. If we use it with Chrous we'll have that. Maybe I can get a small subset indexed in these containers for simple demos/testing.

File: gradle.properties
Comment (RasonJ) [inline gradle.properties:1]: How could we align this config with the dashboard?  It's only slightly annoying each time I compile that it asks for the OS version.

File: README.md
Comment (RasonJ) [inline README.md:1]: I think we should starting adding basic auth, so that we can assume at least some security.

**Pull request merged**